frontend/src/data/Dialect.ts

```diff
@@ -9,6 +9,7 @@ import { fromSerialized } from "./parseReportData";
 export default class Dialect {
   readonly path: string;
   readonly prettyName: string;
+  readonly shortName: string;
```

Comment:
`shortName` is (should be) exactly the same as what the UI calls `path`, I just thought that name made no sense when I extracted out the data for use in the backend too -- we should probably rename `path` rather than adding a duplicate field, or else make `path` be a property that just returns `shortName` or something, but that also could wait for a follow up PR.

Reply:
I can work on this probably in the next PR. This will require many changes in the backend as well as frontend.

Reply:
I clearly got it now and made changes in Dialect.ts and now frontend uses `path` instead of `shortName`.